stacker/blueprints/base.py

```diff
@@ -213,7 +257,20 @@ def resolve_variables(self, variables):
                 logger.debug("Got `None` value for variable %s, ignoring it. "
                              "Default value should be used.", variable.name)
                 continue
-            self.resolved_variables[variable.name] = variable.value
+
```

Comment:
This method is getting a little long.  Might be worth breaking it up, or putting some comments to describe what is happening.